test/unit/resources/org/folio/client/jira/testIssueCreate/issuEditMeta.json

```diff
@@ -142,229 +142,229 @@
         "customId": 10501
       },
       "name": "Development Team",
-      "fieldId": "customfield_10501",
+      "key": "customfield_10501",
       "operations": [
         "set"
       ],
       "allowedValues": [
         {
-          "self": "https://issues.folio.org/rest/api/2/customFieldOption/10304",
+          "self": "https://folio-org.atlassian.net/rest/api/2/customFieldOption/10304",
           "value": "@cult",
           "id": "10304",
           "disabled": false
         },
         {
-          "self": "https://issues.folio.org/rest/api/2/customFieldOption/12226",
+          "self": "https://folio-org.atlassian.net/rest/api/2/customFieldOption/12226",
           "value": "Bama",
           "id": "12226",
           "disabled": false
         },
         {
-          "self": "https://issues.folio.org/rest/api/2/customFieldOption/10308",
+          "self": "https://folio-org.atlassian.net/rest/api/2/customFieldOption/10308",
           "value": "Bienenvolk",
           "id": "10308",
           "disabled": false
         },
         {
-          "self": "https://issues.folio.org/rest/api/2/customFieldOption/10571",
+          "self": "https://folio-org.atlassian.net/rest/api/2/customFieldOption/10571",
           "value": "Concorde",
           "id": "10571",
           "disabled": false
         },
         {
-          "self": "https://issues.folio.org/rest/api/2/customFieldOption/10432",
+          "self": "https://folio-org.atlassian.net/rest/api/2/customFieldOption/10432",
           "value": "Core: Platform",
           "id": "10432",
           "disabled": false
         },
         {
-          "self": "https://issues.folio.org/rest/api/2/customFieldOption/10307",
+          "self": "https://folio-org.atlassian.net/rest/api/2/customFieldOption/10307",
           "value": "EBSCO - FSE",
           "id": "10307",
           "disabled": false
         },
         {
-          "self": "https://issues.folio.org/rest/api/2/customFieldOption/12221",
+          "self": "https://folio-org.atlassian.net/rest/api/2/customFieldOption/12221",
```

Comment:
All of these custom fields should be reviewed carefully, after migration to Jira cloud, most of them have been changed...